src/chapters/angular-wk1/introduction.rst

```diff
@@ -0,0 +1,24 @@
+Why Use JavaScript Libraries
+=============================
+
+We use libraries to make our lives as developers easier. We can focus on solving our specific problems if we don't have to spend time figuring out how information flows throughout our project.
+
+When it comes to web based applications there are two very different places code can exist: in the client's browser, and on the host's server.
+
+A best practice is to separate the code bases for the two different locations. Both sides are required for the web app to work, but they can be thought of as two different projects.
+
+An easier way to think of the two different sides of web app development would be to consider the front end as what the user interacts with, and sees. Consider the back end as the logic, or machinations that the user doesn't need ot think about. Similar to how an old mechanical clock works. The front end would be the face with the 12 numbers, and the two moving hands representing hours, and minutes. The user only needs the clock face to determine the time. The back end for the clock would be the various cogs, wheels, and power of the clock.
+
+In this chapter we will use Angular as our front end JavaScript framework. Angular dictates how we will structure our files, and how the information will flow between the different files, and contains a various number of tools that will help us build out the part of the application our user's will interact with.
```

Comment:
Passive to active changes, and some grammar refinements.  How about:

"In this chapter we will use **Angular** as our front end JavaScript framework. Angular dictates how to structure our files, as well as how the information flows between them. It also contains a number of tools to help us build the part of the application users see in their web browsers."

We need to add a .. index reference for **Angular**.

Reply:
added an index reference, and put in your suggested changes. Emphasized the first usage of Angular.